pages/linux/toolbox-enter.md

```diff
@@ -0,0 +1,12 @@
+# toolbox enter
+
+> Enter a `toolbox` container for interactive use.
+> More information: <https://www.mankier.com/1/toolbox-enter>.
+
+- Enter a `toolbox` container for a different distribution other than the host:
+
+`toolbox enter --distro {{distribution}}`
```

Comment:
Don't you have to specify which container you want to enter? Or does this create a new temporary container?

Reply:
No need to specify which container or toolbox I need to enter. It enters into an existing container/toolbox by default. In case you are running multiple distributions like RHEL, and Fedora. You can specify it in this command. 
[Reference video for toolbox enter](https://user-images.githubusercontent.com/26346867/182399080-014d0f3a-9c8a-47d9-ad5b-9611a484ffe7.webm)
In the above video, host is fedora. So `toolbox enter` works directly as it is host distro, in case of RHEL we can enter a fedora toolbox using `toolbox enter --distro fedora`